tests/src/Common/test_runtime/project.json

```diff
@@ -1,5 +1,6 @@
 {
   "dependencies": {
+    "Microsoft.NETCore.Platforms": "2.0.0-beta-25001-02",
```

Comment:
is this really necessary ? Can't it be in test_dependencies/project.json ?
This json was meant to be only for native assets [ though quite a few managed assets have creeped in]